src/main/kotlin/com/linker/linkerapi/rental/service/RentalService.kt

```diff
@@ -0,0 +1,32 @@
+package com.linker.linkerapi.rental.service
+
+import com.linker.linkerapi.equipment.service.EquipmentService
+import com.linker.linkerapi.rental.entity.Rental
+import com.linker.linkerapi.rental.enums.RentalType
+import com.linker.linkerapi.rental.repository.RentalRepository
+import org.springframework.stereotype.Service
+
+@Service
+class RentalService(
+    private val rentalRepository: RentalRepository,
+    private val equipmentService: EquipmentService
+) {
+    fun createRental(
+        equipmentId: Long,
+        phoneNumber: String,
+        name: String,
+        studentId: Long,
+        rentalTypeStr: String
```

Comment:
```suggestion
        rawRentalType: String
```

전 타입 이름을 붙이는 것 보다는 이렇게 raw라고 붙이는게 더 도움되는 이름인 것 같아요